roles/openshift_logging_elasticsearch/library/openshift_logging_elasticsearch_facts.py

```diff
@@ -0,0 +1,351 @@
+'''
```

Comment:
I think it's alright to keep this within the elasticsearch specific role. It isn't something that would be used anywhere else, so to me it doesn't make sense to move it into a different role or even the installer library